pages/infrastructure_providers-validators/running_full_node.md

```diff
@@ -1,73 +1,76 @@
-# Running a Full Node
+# Run a Full Node
+Running a full node allows your system to participate in a dYdX chain network. The recommended settings and best practices on this page help ensure your node stays healthy and up to date with the network.
 
-### Save your Chain ID in `dydxprotocold` config
+> Code snippets on this page use example values. Replace them with your own. See the [Network Configuration](../infrastructure_providers-network/network_constants.mdx) section of the documentation for network constants and other resources you need to configure a full node.
 
-Save the [chain-id](../infrastructure_providers-network/network_constants.mdx#chain-id). This will make it so you do not have to manually pass in the chain-id flag for every CLI command.
+## Prerequisites
+You need a running, non-validating full node that is connected to network. 
 
-```bash
-dydxprotocold config chain-id $CHAIN_ID
-```
-
-### Getting a Snapshot
+- If you created a system service for your node by following the instructions on the previous page, [Set Up a Full Node](../infrastructure_providers-validators/how_to_set_up_full_node.md), start your node with the following command:
+  ```bash
+  stystemctl start dydxprotocol
+  ```
+- To start your node directly with Cosmovisor, you must include the flag `--non-validating-full-node=true`. Cosmovisor may prompt you to configure additional environment variables or include them in your command.
```

Comment:
can you elaborate on why the flag `--non-validating-full-node=true` is needed. Would be useful for them to know why this flag is being set

Reply:
@affanv14 are you able to provide a short description for Zach?

Reply:
or point him to where he can find descriptions of the flags

Reply:
Here is the description of the flag https://github.com/dydxprotocol/v4-chain/blame/main/protocol/app/flags/flags.go#L78-L83

Reply:
would it make sense to link the flags here to provide more context?

Reply:
I think because we're highlighting this flag, it makes sense to just add some context for it specifically

A list of flags definitely belongs somewhere in the docs, especially since it's hard to find info on the ones listed above (--help returns only a handful of flags)